css/styles.css

```diff
@@ -723,3 +723,7 @@ textarea:focus {
 .burger-icon:hover {
   cursor: pointer;
 }
+
+.formError {
+  color: red;
+}
```

Comment:
- [ ] It may be a good idea to add more styles to your `formError` message to have a great match with the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) (font-family, font-size). 😃 

![contact-form-message](https://user-images.githubusercontent.com/79876959/142672816-f8967d7c-4293-4987-a18c-4e8984bb3882.PNG)

- [ ]  You might want to consider fixing this for the mobile screen.

![contact-form-message-2](https://user-images.githubusercontent.com/79876959/142672911-512acf71-d9bf-455b-bcd3-eaf5ce5ddff6.PNG)